global/settings/types/preferencekeys.h

```diff
@@ -149,6 +149,7 @@
 #define PREF_UI_PIANOROLL_DARK_NOTE_UNSEL_COLOR             "ui/pianoroll/dark/note/unselected/color"
 #define PREF_UI_PIANOROLL_DARK_NOTE_SEL_COLOR               "ui/pianoroll/dark/note/selected/color"
 #define PREF_UI_PIANOROLL_DARK_BG_BASE_COLOR                "ui/pianoroll/dark/background/base/color"
+#define PREF_UI_PIANOROLL_DARK_BG_KEY_HIGHLIGHT_COLOR       "ui/pianoroll/dark/background/keys/hilight/color"
```

Comment:
Please rename this preference to `ui/pianoroll/dark/background/keys/highlight/color` (correcting the spelling of “highlight”).